### PR TITLE
Add the ability to require display fields in seeker

### PR DESCRIPTION
**Pull request by lefcourn**

### CODE REVIEW

File: seeker/views.py [hunk 1]
```diff
@@ -385,7 +397,18 @@ def get_display(self):
...
+        for field, i in self.required_display:
+            # Remove existing instances since this position takes precedence...
+            display_fields = filter(partial(ne, field), display_fields)
+            # If within range, insert the field appropriately...
+            if i <= list_len:
+                display_fields.insert(i, field)
+            # Otherwise, append to the end
+            else:
+                display_fields.append(field)
+        return display_fields
```
Comment (clevelandj) [inline seeker/views.py]: I think we can clean this up a bit, making it more concise and readable. Perhaps something like the following?

```python
display_fields = self.request.GET.getlist('d') or default
display_fields = [f for f in display_fields if f not in self.required_display]
for field, i in self.required_display:
    display_fields.insert(i, field)
return display_fields
```

File: seeker/views.py [hunk 2]
```diff
@@ -160,6 +162,16 @@ class SeekerView (View):
     """
     A list of field/column names to display by default.
     """
+    
+    required_display = []
+    """
+    A list of tuples, ex. ('field name', 0), representing field/column names that will always be displayed (cannot be hidden by the user).
+    """
```
Comment (clevelandj) [inline seeker/views.py]: Let's add some text explaining that the second element of each tuple is the position of the required display field.

File: seeker/views.py [hunk 3]
```diff
@@ -160,6 +162,16 @@ class SeekerView (View):
...
+    
+    @property
+    def required_display_fields(self):
+        for field_name, i in self.required_display:
+            yield field_name
```
Comment (clevelandj) [inline seeker/views.py]: Using a generator seems a bit overkill here. Maybe switch to:
```python
@property
def required_display_fields(self):
    return [t[0] for t in self.required_display]
```

Comment (lefcourn) [inline seeker/views.py]: Is that faster? I would have thought they were doing basically the same thing

File: seeker/views.py (hunk 1 above)
Comment (clevelandj) [inline seeker/views.py]: Having a duplicate field would be entirely user error, though, right? Couldn't the user fix that by simply removing the duplicate from their config?

Comment (lefcourn) [inline seeker/views.py]: I think, one situation at least, would be if the user reorders the field. I think this will prevent them from doing that.

**Pull request merged**